dashboard/styles.css

```
@@ -0,0 +1,1333 @@
+
+#detail-toggle[data-on="1"] {
+    background: var(--ncstate-red);
+    border-color: var(--ncstate-red);
+    color: #fff;
+}
+
+.toggle-dot {
+    width: 7px;
+    height: 7px;
+    border-radius: 50%;
+    background: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)